VEuPathDB/web-monorepo
# 105-map-logo

**Before**
![image](https://user-images.githubusercontent.com/365139/234164024-f44da650-d68e-4364-a2c3-2e3f6fa1bedf.png)

**After**
![image](https://user-images.githubusercontent.com/365139/234163962-223ce8b7-c62a-4037-8efe-46fd88b65d55.png)

### Description

Fixes #105

This PR cleans up some files from the `webapp` directory of site packages, and copies all deployable assets to `dist` when the `bundle` scripts run. It also updates `siteInformationProps` passed to the map app.

I removed a lot of files that erroneously got copied into this repo, in this PR. The most relevant commit is https://github.com/VEuPathDB/web-monorepo/pull/109/commits/b5680930236d720a7d8ce35425b48f3fffa320b0, which updates the `/maps` route to include better `siteInformationProps` values.


### Testing

The best way to fully test this PR is to:

1. Build a dev site
2. Run `yarn nx bundle:dev $sitePackage`
3. Run `rsync -rvz  packages/sites/$sitePackage/dist/* $server:$targetWebappDir`

I've done this already with https://dfalke-b.clinepidb.org. If you go to https://dfalke-b.clinepidb.org/ce.dfalke/app/maps/DS_d6a1141fbf, and create a new analysis, you will see the clinepi logo in the map header. You can also use that site as a backend with a local dev site, to see the changes.
